packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -203,7 +205,11 @@ export const CreateOrEditOutOfOfficeEntryModal = ({
                 />
                 {!hasTeamPlan && (
                   <div className="mx-2" data-testid="upgrade-team-badge">
-                    <UpgradeTeamsBadge />
```

Comment:
1. Is there a reason you duplicated code from UpgradeTeamsBadge here?
2. the code snippet below is in `UpgradeTeamsBadge` but not here
```
const { hasPaidPlan } = useHasPaidPlan();

  if (hasPaidPlan) return null;
```

Reply:
Yeah, actually, there's no need to duplicate the code here. I just noticed that in UpgradeTeamsBadge, we check whether the user has a paid plan or not. It is always true if the user has no teams and a premium username. However, since the badge is meant to suggest an upgrade to a team plan, we need to check whether they have a team plan or not. Fixed it